install/environment.root.yml

```diff
@@ -1,18 +1,10 @@
 name: zarp
 channels:
+  - bioconda
   - conda-forge
   - defaults
 dependencies:
-  - graphviz=2.40.1
-  - jinja2=3.0.3
   - mamba=0.16.0
```

Comment:
I think (following the instructions in `README.md` and @AngryMaciek's comment earlier) that this should be removed from _all_ environment files and instead installed ahead of everything else.

Reply:
These should be installed in the main _zarp_ env, ahead of all the per-rule envs. Here we are good: `environment.root.yml` is a top lvl env.

Reply:
`mamba` still needs to be removed, though.

Reply:
Had it removed. Probably got lost in the different commits.